kahuna/public/js/main.js

```diff
@@ -125,7 +125,7 @@ kahuna.run(['$log', '$rootScope', 'mediaApi', function($log, $rootScope, mediaAp
                     authAndRedirect(loginUriTemplate);
                 } else {
                     // Couldn't extract a login URI, die noisily
-                    throw error;
+                    throw new Error('Failed to redirect to auth, no login URI found');
```

Comment:
Errors thrown in a Promise chain should be surfaced on the window by Angular anyway, so I think that would be superfluous.